CMakeLists.txt

```diff
@@ -242,7 +242,7 @@ endif()
 #set(HDF5_PREFER_PARALLEL True)
 if (NETCDF)
   find_package(NetCDF REQUIRED)
-  target_include_directories(SeisSol-common-properties INTERFACE ${NetCDF_INCLUDE_DIRS})
+  target_include_directories(SeisSol-common-properties INTERFACE ${NetCDF_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR}/src/Geometry/)
```

Comment:
Obsolete?

Reply:
You're right, specifying that directory is redundant.